src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -5462,16 +5475,37 @@ def test_unsupported_halo_read_access():
     # call our method
     with pytest.raises(GenerationError) as err:
         _ = loop._halo_read_access(stencil_arg)
-    assert ("Loop bounds other than cell_halo and ncells are currently "
+    assert ("Loop bounds other than 'cell_halo' and 'ncells' are currently "
             "unsupported for kernels with stencil accesses. Found "
             "'inner'." in str(err.value))
 
 
+def test_dynglobalsum_unsupported_argument():
+    ''' Check that an instance of the DynGlobalSum class raises an
+    exception if an unsupported argument type. '''
```

Comment:
"if an unsupported type....??"

Reply:
Oops. Should be "for an unsupported argument type". Changed.